docs/protocol/ISM/multisig-ISM.mdx

```diff
@@ -1,3 +1,40 @@
 # Multisig ISM
 
 The `MultisigISM` is one of the most commonly used ISM types. These ISMs verify that `m` of `n` [Validators](../agents/validators.mdx) have attested to the validity of a particular interchain message.
+
+## Multisig Module Types
+
+Based on the following module types, the relayer will encode the appropriately formatted metadata:
+
+```solidity
+enum Types {
+    ...
+    MERKLE_ROOT_MULTISIG,
+    MESSAGE_ID_MULTISIG,
+    ...
```

Comment:
because this is numeric, I would either include the preceding fields or include `= X`